versioned_docs/version-v3.4/testing/config.md

```diff
@@ -71,7 +71,7 @@ export interface TestingConfig extends JestConfig {
    * The following values are accepted:
    * - "new" - enables the "new" headless mode, starting with Chrome 112
    * - `true` - enables the "old" headless mode, prior to Chrome 112
-   * - `false` - enables the "headful" mode
```

Comment:
This was actually intentional - I wanted to keep with the same language here that the[ Chrome team used in their announcement for the flag](https://developer.chrome.com/articles/new-headless/#whats-new-in-headless) for searchability reasons (the thought being someone reads it here for the first time, then google 'chrome headful').  Would you mind reverting this change?

Reply:
So weird. Puppeteer uses it too.